homeassistant/components/media_player/philips_js.py

```diff
@@ -117,14 +122,22 @@ def source_list(self):
         """List of available input sources."""
         return self._source_list
 
+    def _select_body(self, ccid):
+        return {"channelList": {"id": "alltv"}, "channel": {"ccid": ccid}}
```

Comment:
This is protocol specific information. This should be part of the `ha-philipsjs` lib

Reply:
Ok, thanks. I will try to update the lib.